lte/gateway/c/session_manager/SessionState.h

```diff
@@ -26,6 +26,7 @@
 #include "SessionCredit.h"
 #include "Monitor.h"
 #include "ChargingGrant.h"
+#define SESSION_THROTTLE_CNT 3
```

Comment:
Should we add a TODO here to have it configured in `sessiond.yml` ?

Reply:
Resolved